src/controllers/albums.js

```diff
@@ -0,0 +1,98 @@
+const db = require('../db/index')
```

Comment:
This whole file needs formatting as your indentation is 4 whereas your prettier config is 2